src/dataExplorer/components/Schema.scss

```diff
@@ -62,39 +62,45 @@
 }
 
 .field-selector--list-item,
-.field-selector--list-item--selectable,
-.tag-selector-value--list-item,
-.tag-selector-value--list-item--selectable {
+.tag-selector-value--list-item {
   padding: $cf-space-2xs 0 $cf-space-2xs $cf-space-2xs;
   white-space: nowrap;
   overflow: hidden;
   text-overflow: ellipsis;
 }
 
-.tag-selector-key--list-item {
-  padding: $cf-space-3xs 0;
+.field-selector--list-item--selectable,
+.tag-selector-value--list-item--selectable {
   white-space: nowrap;
   overflow: hidden;
-  text-overflow: ellipsis;
-}
 
-.field-selector--list-item--wrapper,
-.tag-selector-value--list-item--wrapper {
-  width: fit-content;
+  .cf-text-block {
+    text-overflow: ellipsis;
+    width: fit-content;
+    font-family: $cf-code-font;
+    font-weight: 400;
+    background-color: inherit;
+    color: $cf-white;
+    transition-property: color;
+
+    &:hover {
+      cursor: pointer;
+      color: #7ce490; // color suggested in https://github.com/influxdata/ui/issues/4905
```

Comment:
why are we using a color not defined in clockface and not changing the color in monaco to match our theming instead? all for the standardization, just seems like this is the wrong direction of that equation?

Reply:
@drdelambre It's a good point! I thought of it and not sure if it's too big of a change if I change the color setting in Monaco config for entire UI. I'd prefer to change the color in Monaco instead of the other way around to avoid too many hard coded hex code in our code base. What do you think @juliajames12?

Here are the comparison of colors FYI

## Using suggested color #7ce490
<img width="1128" alt="Screen Shot 2022-06-28 at 10 37 37 AM" src="https://user-images.githubusercontent.com/14298407/176223705-a167e133-48ed-4e43-94d3-bb52e95c540d.png">

## Using clockface color #9bf445 ($c-krypton)
<img width="1137" alt="Screen Shot 2022-06-28 at 10 36 54 AM" src="https://user-images.githubusercontent.com/14298407/176223739-d2fea8d3-20b2-4fa1-b736-27fcae3f15bc.png">